opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -0,0 +1,111 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=protected-access
+
+from logging import getLogger
+from re import match
+
+_logger = getLogger(__name__)
+
+
+class View:
+    def __init__(
+        self,
+        instrument_type=None,
+        instrument_name=None,
+        meter_name=None,
+        meter_version=None,
+        meter_schema_url=None,
+        name=None,
+        description=None,
+        attribute_keys=None,
+        extra_dimensions=None,
+        aggregation=None,
+        exemplar_reservoir=None,
```

Comment:
Made view a dataclass :v: and added documentation for every parameter and for the class. Included in the documentation information for the user so that the purpose of every attribute can be identified.